blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -439,7 +443,7 @@ export const actions = {
                                 return { ...data,
                                     id: event.jenkins_object_id, // make sure the runId is set so we can find it later
                                     state: 'RUNNING', // This is a horrible hack due to issues with QUEUED status
-                                    result: event.job_run_status,
+                                    result: 'UNKNOWN',
```

Comment:
yes I would like to know. The result should NOT however be "queued", but the state should be if it really was queued (see horrible hack above)